6.js

```diff
@@ -0,0 +1,31 @@
+function pathTo4chan (jsonObject) {
```

Comment:
I think the idea of this exercise is to create a function that will find the path to any value, however many levels deep in the object. I'd rewrite it to take a JSON object and any value and return the path to that value.